server/helper.ts

```diff
@@ -183,3 +184,17 @@ function compareWithWildcard(a: string, b: string) {
 	const re = new RegExp(`^${user_regex}$`, "i"); // case insensitive
 	return re.test(b);
 }
+
+function catch_to_error(prefix: string, err: any): Error {
```

Comment:
That thing is bonkers, but TS yells at me if I shove the error in a string literal from a promise catch block... as that doesn't guarantee that I'm actually looking at an Error (the interface)...

So you need to type assert this and I couldn't figure out how to do it without the code in this function... and I'm not going to inline this in every catch block